kafka_consumer/datadog_checks/kafka_consumer/kafka_consumer.py

```diff
@@ -148,14 +148,14 @@ def report_consumer_offsets_and_lag(self, contexts_limit):
                 self.log.warning(msg, consumer_group, topic, partition)
                 self.client.request_metadata_update()  # force metadata update on next poll()
 
-    @AgentCheck.metadata_entrypoint
-    def collect_broker_metadata(self):
-        version_data = [str(part) for part in self.client.collect_broker_version()]
-        version_parts = {name: part for name, part in zip(('major', 'minor', 'patch'), version_data)}
-
-        self.set_metadata(
-            'version', '.'.join(version_data), scheme='parts', final_scheme='semver', part_map=version_parts
-        )
+    # @AgentCheck.metadata_entrypoint
```

Comment:
To reviewer: the new Confluent library does not actually collect the Kafka version metadata.